CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+### 3.0.2
+
+An automatically generated list of changes can be found on GitHub at: [3.0.2 Release](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v3.0.2)
+
+A curated list of changes can be found on the [Releases](http://docs.nginx.com/nginx-ingress-controller/releases/) page on the NGINX Documentation website.
+
+### ic-version
+
+An automatically generated list of changes can be found on GitHub at: [ic-version Release](https://github.com/nginxinc/kubernetes-ingress/releases/tag/vic-version)
+
+A curated list of changes can be found on the [Releases](http://docs.nginx.com/nginx-ingress-controller/releases/) page on the NGINX Documentation website.
+
```

Comment:
```suggestion
```

This shouldn't be here (there might be something wrong the script? We can check after this release)